tests/pact/try.repl

```diff
@@ -0,0 +1,51 @@
+(begin-tx)
+
+(module try-module GOVERNANCE
+
+  "Test the try/handle-based control flow primitives"
+
+  (defcap GOVERNANCE ()
+    (enforce false "non-upgradeable test"))
+
+  (defun t:bool ()
+    (try false (enforce (= "emily" "emily") "failure")))
+
+  (defun u:bool ()
+    (try false (enforce (= "will" "stuart") "failure")))
```

Comment:
perhaps in this test do a different type, which should also prove that the code actually blows nonlocally
```
  (try "default" (let ((r "broken")) (enforce false "will fail") r))
```